API.md

```diff
@@ -1,13 +1,18 @@
 REST API Documentation :
 
-* Secure your API with the `rest_api_password` setting. This password is shared across all bots. If the setting isn't set, it will be generated and saved in the config file.
-* Request an access token with a `POST` to `/api/bot/{name}/auth`, the request body must be `rest_api_password`. This token must be placed in the `X-PGB-ACCESS-TOKEN` header for further use of the API.
-* Bot end-point : 
+* Secure your API by putting a secure and random password in the `rest_api_password` setting in your config.properties file. This password is shared across all bots. If the configuration key is not set, it will be generated and saved in the config file.
```

Comment:
Paging @Peyphour -> how does this work with the JSON files? Also I did notice if it was empty, it didn't autofill in the configuration file :o